### DISCUSSION TITLE
易支付回调问题

### DISCUSSION THREAD

**Discussion by kaze-desu** [Baiyuetribe/kamiFaka#69] [Bug反馈]
正式版中易支付是否存在回调问题，使用易支付时无法正常回调，补单提示success，但是发卡仍然为未支付状态

**Comment by kaze-desu** [comment]
怀疑是站点配置问题，但是用码支付的时候就是正常的
另外麻瓜支付也有一样的问题：
[![gd90wq.png](https://z3.ax1x.com/2021/05/11/gd90wq.png)](https://imgtu.com/i/gd90wq)

**Reply by Baiyuetribe** [reply]
没有设置回调域名，管理后台，网站设置特别标注了

**Reply by kaze-desu** [reply]
> 没有设置回调域名，管理后台，网站设置特别标注了

web_url是填写了的，内容为网站主域

**Comment by xuelang113** [comment]
如果你直接拉取这个镜像：baiyuetribe/kamifaka:latest
那易支付回调确实是失败的，但是我拉取这个镜像：baiyuetribe/kamifaka:v1.8
易支付就成功了。
